src/components/AccountActivityDataCellContractCallTxEvent.vue

```diff
@@ -6,11 +6,17 @@
   <app-chip size="sm">
     {{ activity.payload.tx.function }}
   </app-chip>
+  <app-chip
+    v-if="activity.payload.tx.function === 'add_vote'"
```

Comment:
why do we only want to show this for this function? I'd propose to show it wherever amount !== 0

Reply:
Yeah, makes sense. Thanks for the suggestion. I checked it and didn't find any issue with it. Improved.